### DISCUSSION TITLE
No Klingon Spaceframes

### DISCUSSION THREAD

**Discussion by SMG001** [bcholmes/StarTrek2d20#108] [General]
So apparently, there's a new problem. I was trying to make a Klingon Starship, because at one point in my campaign, my players will have to command one, but when I went to select a spaceframe for the vessel, there was none, so I came here to report this.

**Comment by bcholmes** [comment]
That's not a happy-making thing. I've pushed a fix. Thanks for reporting that!

**Reply by SMG001** [reply]
No problem!
